pyiso/base.py

```diff
@@ -316,7 +316,8 @@ def unzip(self, content):
             # have zipfile
             z = zipfile.ZipFile(filecontent)
         except zipfile.BadZipfile:
-            LOGGER.error('%s: unzip failure for content:\n%s' % (self.NAME, content))
+            LOGGER.error('%s: unzip failure for content beginning:\n%s' % (self.NAME, str(content)[0:100]))
```

Comment:
At first I thought this was unsafe behaviour. Today I learned that slicing potentially out-of-bound positions isn't a big deal in Python :+1: